source/ntbs-reporting.sqlproj

```diff
@@ -117,7 +117,6 @@
     <Folder Include="dbo\Views\Authorization" />
     <Folder Include="dbo\Views\Reference Data" />
     <Folder Include="dbo\Tables\Power BI Reporting" />
-    <Folder Include="dbo\Stored Procedures\Power BI Reporting" />
```

Comment:
The changes to the project file don't look right to me, this shouldn't be removed

Reply:
My IDE seems to have done a few strange things - Should be fixed now